build.sbt

```diff
@@ -202,6 +205,25 @@ def currentGitHash(dir: File): String = {
   ).!!.trim
 }
 
+lazy val setupHail = taskKey[Unit]("Set up Hail")
+ThisBuild / setupHail := {
+  Seq(
+    "/bin/bash",
+    "-c",
+    s"git clone -b ${hailVersion.value} https://github.com/hail-is/hail.git;" +
+    "source $(conda info --base)/etc/profile.d/conda.sh &&" +
+    "conda create -y --name hail &&" +
+    "conda activate hail --stack &&" +
+    s"make -C hail/hail SCALA_VERSION=${scalaVersion.value} SPARK_VERSION=${sparkVersion.value} install-deps shadowJar wheel &&" +
+    s"pip install --no-deps hail/hail/build/deploy/dist/hail-${hailVersion.value}-py3-none-any.whl"
+  ) !
+}
+
+lazy val teardownHail = taskKey[Unit]("Tear down Hail")
+ThisBuild / teardownHail := {
```

Comment:
I think this should be `cleanHail` or `uninstallHail`. `teardown` usually refers to the decomissioning of an active process.